ooniprobe/Model/Database/Measurement.mm

```diff
@@ -96,22 +96,30 @@ -(void)deleteObject{
     [self remove];
 }
 
-- (void)getExplorerUrl:(void (^)(NSString *))measurement_url {
-    NSString *path = [NSString stringWithFormat:@"https://api.ooni.io/api/v1/measurements?report_id=%@&input=%@", self.report_id, self.url_id.url];
+-(void)getExplorerUrlSuccessCallback:(void (^)(NSString*))success errorCallback:(void (^)(NSError*))error {
```

Comment:
Nit: I think `error` is an instance on error. Better calling the variable `errorcb` (and `successcb` the success callback.)